pkg/spec/spec.go

```diff
@@ -73,6 +74,14 @@ func CreateConfigToOCISpec(config *CreateConfig) (*spec.Spec, error) { //nolint
 		g.AddAnnotation(key, val)
 	}
 	g.SetRootReadonly(config.ReadOnlyRootfs)
+	if config.Hostname == "" {
+		if config.NetMode.IsHost() {
+			config.Hostname, err = os.Hostname()
+			if err != nil {
+				return nil, errors.Wrap(err, "unable to retrieve hostname")
+			}
+		}
+	}
```

Comment:
I think this is OK as is, but thought I'd ask just in case.  If by chance config.Hostname is still equal to "" at this point, do we want to continue on as we do now or raise an "unable to determine hostname" type of error?

Reply:
The Hostname will get set to the UUID of the container, I believe if it is still ""/

Reply:
Yeah, if it's still "" we will override it with the container's UUID in libpod.

Reply:
Works for me, thanks for the verification.